addons/main/XEH_postInit.sqf

```diff
@@ -515,4 +515,8 @@ GVAR(addingPlate) = false;
 },
 [DIK_T, [false, false, false]], false] call CBA_fnc_addKeybind;
 
+if (GVAR(enableBleedoutTimerRegen)) then {
+    player call FUNC(regenBleedout);
+};
```

Comment:
make sure this does not error if ace medical is loaded. Neither the variable nor the function are defined then.